### PR TITLE
11.0 mig l10n nl autoinstall

### PR DESCRIPTION
**Pull request by daramousk**

### CODE REVIEW

**Comment by daramousk** [comment]
@hbrunn Travis crashes for some reason.

**Comment by hbrunn** [comment]
we're using an outdated `.travis.yml` here - get https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml and change what needs changing, add this as a separate commit please

File: l10n_nl_autoinstall/__manifest__.py [hunk 1]
```diff
@@ -1,10 +1,9 @@
-# -*- coding: utf-8 -*-
 # Copyright 2015-2018 Therp BV <https://therp.nl>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Dutch chart of accounts - autoinstall",
-    "version": "10.0.1.0.1",
-    "author": "Therp BV",
+    "version": "11.0.0.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
```
Comment (hbrunn) [inline l10n_nl_autoinstall/__manifest__.py]: in this repo, it's actually ours

**Review by hbrunn — CHANGES_REQUESTED** [review]

File: l10n_nl_autoinstall/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2015-2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Dutch chart of accounts - autoinstall",
+    "version": "11.0.0.0.0",
```
Comment (hbrunn) [inline l10n_nl_autoinstall/__manifest__.py]: standard version number should be `11.0.1.0.0`

**Comment by daramousk** [comment]
@hbrunn Fixed, I would appreciate this merged when possible because I need travis to work.

**Review by hbrunn — APPROVED** [review]

**Pull request merged**